Lombiq.Tests.UI/Extensions/FrontendUITestContextExtensions.cs

```diff
@@ -13,6 +13,34 @@ namespace Lombiq.Tests.UI.Extensions;
 
 public static class FrontendUITestContextExtensions
 {
+    public const string FrontendPseudoTenantName = "!Frontend";
+
+    /// <summary>
+    /// Navigates to the backend <see cref="Uri"/> returned by <see
+    /// cref="FrontendOrchardCoreUITestExecutorConfigurationExtensions.GetFrontendAndBackendUris"/> and presents it as
+    /// switching to the default tenant.
+    /// </summary>
+    /// <remarks><para>
+    /// If the backend URL has not been initialized to something else (e.g. using a custom URL prefix), this is
+    /// equivalent to using <see cref="UITestContext.SwitchCurrentTenantToDefault"/>. Even so, this method should be
+    /// used for clarity when applicable.
+    /// </para></remarks>
+    public static void SwitchToBackend(this UITestContext context) =>
+        context.SwitchCurrentTenant(
+            tenantName: null,
+            context.Configuration.GetFrontendAndBackendUris().BackendUri);
+
+    /// <summary>
+    /// Navigates to the frontend <see cref="Uri"/> returned by <see
+    /// cref="FrontendOrchardCoreUITestExecutorConfigurationExtensions.GetFrontendAndBackendUris"/> and presents it as
+    /// switching to a tenant named <see cref="FrontendPseudoTenantName"/> which is not a real Orchard Core tenant so
+    /// this information can only be used for information.
```

Comment:
What do you mean can be used only for information? BTW there's a duplicate of "information" there.

Reply:
That you can't use it in methods that take a tenant name.